espanol/poeta/poeta.md

```diff
@@ -0,0 +1 @@
+Y al observar a su alrededor, el intrigado hombre penso que los hechos ocurridos darian fruto a una hermosa historia.  Pero, para recodar a su hermana, insistió en que tenia que ser un poema.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.